### PR TITLE
SPV-84 Optimize DB initialization

### PR DESCRIPTION
**Pull request by andrii-kl**

### CODE REVIEW

File: nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java [hunk 1]
```diff
@@ -95,7 +95,11 @@ public void addAll(List<VeriBlockBlock> blocks) throws SQLException {
 
         List<StoredVeriBlockBlock> storedBlocks = convertToStoreVeriBlocks(listToStore);
 
-        blockStore.put(storedBlocks);
+        try {
+            blockStore.put(storedBlocks);
+        } catch (Exception ex){
+            System.out.printf("" + ex.getMessage(), ex);
```
Comment (PCasafont) [inline nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java]:
```suggestion
            logger.error("Error while storing blocks", ex);
```
And maybe rethrow as an illegal state exception?

File: nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java [hunk 2]
```diff
@@ -125,7 +129,12 @@ public void addAll(List<VeriBlockBlock> blocks) throws SQLException {
             BigInteger work = blockWorks.get(veriBlockBlock.getPreviousBlock().toString());
             //This block is from fork, our Blockchain doesn't have this previousBlock.
             if(work == null){
-                continue;
+                StoredVeriBlockBlock storedVeriBlockBlock = blockStore.get(veriBlockBlock.getPreviousBlock());
+                if(storedVeriBlockBlock == null) {
```
Comment (PCasafont) [inline nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java]:
```suggestion
                if (storedVeriBlockBlock == null) {
```
Doesn't the editor config work on your IDE?

File: nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java (hunk 1 above)
Comment (andrii-kl) [reply nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java]: Aah... I added this catch for debugging.

File: nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java (hunk 2 above)
Comment (andrii-kl) [reply nodecore-spv/src/main/java/veriblock/service/impl/Blockchain.java]: I will check it.

**Pull request merged**